test/helpers/cons.go

```diff
@@ -171,7 +171,7 @@ const (
 	KubectlPolicyNameLabel      = k8sConst.PolicyLabelName
 	KubectlPolicyNameSpaceLabel = k8sConst.PolicyLabelNamespace
 
-	CiliumStableVersion      = "v1.1.4"
+	CiliumStableVersion      = "v1.1"
```

Comment:
Nice, this makes things easier!

Reply:
Don't get too excited, check the commit message.